tests/testthat/test-sptf-textureclass.R

```diff
@@ -0,0 +1,23 @@
+# test bulk density functions
+require(testthat)
+
+test_that("USDA texture classification", {
+  
+  # input parameters
+  dt <- data.table(
+    A_CLAY_MI = c(10, 30),
+    A_SILT_MI = c(20, 20),
+    A_SAND_MI = c(60, 50)
```

Comment:
The first example does not add up to 100. If there are more possible return values for this function it would be good to increase the number of test cases.